src/EventStore.Core/DataStructures/ProbabilisticFilter/IProbabilisticFilter.cs

```diff
@@ -4,11 +4,11 @@ namespace EventStore.Core.DataStructures.ProbabilisticFilter {
 	//qq suspicious that we're not accessing the implementations through these interfaces
 	public interface IProbabilisticFilter<in TItem> {
 		void Add(TItem item);
-		bool MayExist(TItem item);
+		bool MightExist(TItem item);
```

Comment:
I like the name but `CouldContain` might be closer to other names?

Reply:
Oh yeah, that’s nice 👍